### ISSUE TITLE
Feature Request: Merge Lycoris/LoHa to Checkpoint

### ISSUE THREAD

**Issue by kieranxsomer** [hako-mikan/sd-webui-supermerger#68]
Literally noticed some earlier Lycoris we made were still in the old Lora format, and COULD merge to a checkpoint.  Newer formats aren't supported yet, and that's valid!  But there's a few of my created lycoris files i'd love to be able to fuse and bind to checkpoints for the fun of it. 

When you have time, :)

**Comment by hako-mikan** [comment]
The structure of loha is special and I expect the author of Loha to create it.
Is the loha merge not in the LyCORIS author's repository?

**Comment by AshtakaOOf** [comment]
Here's the script https://github.com/KohakuBlueleaf/LyCORIS/blob/main/tools/merge.py

**Comment by AshtakaOOf** [comment]
This was closed as completed but I don't see any commits related to this issue?

**Comment by ScrapWare** [comment]
Whatever used on generate, Whatever can merge. However, updown is different so use merged weight in the first. C3Lier can stucking every updown then no problem supermerger is continuous single merger no stucking with LyCORIS any algo.

LoHa needed calculate hada as weights, Example is 1a@1b 2a@2b 1a1b*2a2b

maybe can joint 1a and 2a through matmul? I don't tried it I don't known it could stucking those converted updown.

Nothing worry if used webui builtin API maybe. LyCORIS updown is not stuckable with no conversion to weights when annoying slow of merge. Reccomended L3Lier LoCon extends full layer.

**Comment by ScrapWare** [comment]
Oops, Then can stucking converted weight. I don't know ia3 and some specifications. I'll not recommended LyCORIS because can't fastest torch.add merge on non calc every up and down.

**Comment by ScrapWare** [comment]
Solve these networks with both torch.add value stack and torch.cat dim stack are make fastest merger too,,, and also I don't understand use SVD. This issues mark closed then nothing to do. I haven't free of publish time and haven't means of publish at forever. I hate maintaining🙃

![Image](https://github.com/user-attachments/assets/27623d11-b9fe-40e5-a0c2-ea13501fdee7)
